### PR TITLE
perf: Use list append when string building

### PR DESCRIPTION
**Pull request by MegaAnda123**
## Description
*Include a description of the change done*

# Why
*The motivation for the change (if applicable)*

# How
*How the change is implemented (if it's not self evident in the diff)*

Close: #XXX

# Checklist:
Before submitting this PR, please make sure:

- [ ] I am complying with the [contributing](https://equinor.github.io/completor/contribution_guide) doc
- [ ] My code builds clean without any errors or warnings
- [ ] I have added/extended tests that prove my fix is effective or that my feature works
- [ ] I have made corresponding changes to the documentation, and it builds correctly

### CODE REVIEW

**Review by Aleksander-Karlsson — APPROVED** [review]
LGTM, missing some docu in the PR description though

File: completor/create_output.py [hunk 1]
```diff
@@ -572,3 +534,42 @@ def _connect_lateral(well_name: str, lateral: Lateral, top: pd.DataFrame, well:
     out_segment = layer_to_connect.at[idx, Headers.START_SEGMENT_NUMBER]
     lateral.df_tubing.at[0, Headers.OUT] = out_segment
     return lateral.df_tubing
+
+
+def _format_density_activated_recovery2(data: str) -> str:
```
Comment (Aleksander-Karlsson) [inline completor/create_output.py:539]:
```suggestion
def _format_density_activated_recovery(data: str) -> str:
```

File: completor/create_output.py [hunk 2]
```diff
@@ -572,3 +534,42 @@ def _connect_lateral(well_name: str, lateral: Lateral, top: pd.DataFrame, well:
...
+        "-- and gas volume fractions at downhole condition.\n"
+        "-- The value of Cv is adjusted according to the segment length and the number of\n"
+        "-- devices per joint. The constriction area varies according to values of\n"
+        "-- volume fractions.\n"
+        f"{'-' * 100}\n\n\n"
+    )
+    return header + data + "\n\n\n\n"
+
+
+def _format_autonomous_inflow_control_valve2(data: str) -> str:
```
Comment (Aleksander-Karlsson) [inline completor/create_output.py:560]:
```suggestion
def _format_autonomous_inflow_control_valve(data: str) -> str:
```

File: completor/create_output.py [hunk 3]
```diff
@@ -163,21 +163,30 @@ def format_output(well: Well, figure_name: str | None = None, paths: tuple[str,
...
+    if print_valve:
+        output.append(f"{Keywords.WELL_SEGMENTS_VALVE}{print_valve}\n/\n\n\n")
+    if print_inflow_control_device:
+        output.append(f"{Keywords.INFLOW_CONTROL_DEVICE}{print_inflow_control_device}\n/\n\n\n")
+    if print_autonomous_inflow_control_device:
+        output.append(f"{Keywords.AUTONOMOUS_INFLOW_CONTROL_DEVICE}{print_autonomous_inflow_control_device}\n/\n\n\n")
+    if print_inflow_control_valve:
+        output.append(f"{Keywords.WELL_SEGMENTS_VALVE}{print_inflow_control_valve}\n/\n\n\n")
+    if print_density_activated_recovery:
+        output.append(_format_density_activated_recovery2(print_density_activated_recovery))
```
Comment (Aleksander-Karlsson) [inline completor/create_output.py:185]:
```suggestion
        output.append(_format_density_activated_recovery(print_density_activated_recovery))
```

File: completor/create_output.py [hunk 4]
```diff
@@ -163,21 +163,30 @@ def format_output(well: Well, figure_name: str | None = None, paths: tuple[str,
...
+    if print_autonomous_inflow_control_valve:
+        output.append(_format_autonomous_inflow_control_valve2(print_autonomous_inflow_control_valve))
```
Comment (Aleksander-Karlsson) [inline completor/create_output.py:187]:
```suggestion
        output.append(_format_autonomous_inflow_control_valve(print_autonomous_inflow_control_valve))
```

File: completor/create_output.py (hunk 2 above)
Comment (MegaAnda123) [reply completor/create_output.py:560]: Will be done in another pr.

**Pull request merged**